src/overlays/actors/ovl_En_Ta/z_en_ta.h

```diff
@@ -9,31 +9,60 @@
 struct EnTa;
 
 typedef void (*EnTaActionFunc)(struct EnTa*, PlayState*);
-typedef void (*EnTaUnkFunc)(struct EnTa*);
+typedef void (*EnTaBlinkFunc)(struct EnTa*);
+typedef void (*EnTaAnimFunc)(struct EnTa*);
+
+typedef enum {
+    /* 0 */ ENTA_OTHER, // Child era
```

Comment:
```suggestion
    /* 0 */ ENTA_CHILD_ERA,
```
Any reason not to call it the Child era type?

Reply:
This came from a printf that translated to "other Talon", but it's not actually used anywhere. EnTa_Init only has cases for the other two types and the default case. Now that I checked this, child era Talon actors actually use `0xFFFF` as the param, not zero. 

But yeah, I think changing this to CHILD_ERA is fine as that describes the other states better.